### ISSUE TITLE
Error when 'export as namespace foo' occurs at EOF with no semicolon

### ISSUE THREAD

**Issue by DanielRosenwasser** [Microsoft/TypeScript#11545]
```ts
declare namespace thing {
    export interface ThingStatic {
        hello: string;
        world: string;
    }
}
declare var thing: thing.ThingStatic;

export = thing;

export as namespace GlobalThing
```

Expected: no error
Actual: `';' expected.`

![image](https://cloud.githubusercontent.com/assets/972891/19293582/8387f354-8fda-11e6-8f08-30374a979b6d.png)

**Comment by mhegazy** [comment]
thanks @arusakov!
